x-pack/test/security_solution_cypress/package.json

```diff
@@ -16,7 +16,7 @@
     "cypress:detection_engine:exceptions:run:ess": "yarn cypress:ess --spec './cypress/e2e/detection_response/detection_engine/exceptions/**/*.cy.ts'",
     "cypress:ai_assistant:run:ess":"yarn cypress:ess --spec './cypress/e2e/ai_assistant/**/*.cy.ts'",
     "cypress:run:respops:ess": "yarn cypress:ess --spec './cypress/e2e/(detection_response)/**/*.cy.ts'",
-    "cypress:investigations:run:ess": "yarn cypress:ess --spec './cypress/e2e/investigations/**/*.cy.ts'",
+    "cypress:investigations:run:ess": "yarn cypress:ess --spec './cypress/e2e/investigations/alerts/alert_table_controls.cy.ts'",
```

Comment:
Shall we revert this change?

Reply:
yes I will revert before merging, they were just here to facilitate the job of the flaky test runner! :)